src/main/resources/db/changelog/changes/04-create-users-roles-table.yaml

```diff
@@ -0,0 +1,28 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-users-roles-table
+      author: mateacademy
+      changes:
+        - createTable:
+            tableName: users_roles
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    primaryKey: true
+                    primaryKeyName: pk_user_role
```

Comment:
remove
```
primaryKey: true
primaryKeyName: pk_user_role
```

Reply:
The new version of creating the users_roles table creates a unique composite key for each table record. It seems to be better, if not - correct me

Reply:
Whar do you mean the new version of creating the users_roles table? 
For my opinion, composite key is used for a little different approach

https://www.baeldung.com/jpa-many-to-many